src/index.html

```diff
@@ -11,7 +11,168 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="field">
+        <legend class="legends">Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            autocomplete="off"
+            name="surname"
+            class="inputs"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            autocomplete="off"
+            name="name"
+            class="inputs">
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            value="12"
+            name="oldAreYou"
+            min="1"
+            max="100"
+            class="inputs">
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            class="inputs">
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="iAcceptAgreement"
+            class="inputs">
+        </label>
+      </fieldset>
+
+      <fieldset class="field">
+        <legend class="legends">Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            placeholder="email@example.com"
+            name="email"
+            class="inputs"
+            required>
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+            minlength="3"
+            maxlength="10"
+            class="inputs">
```

Comment:
```suggestion
          <input
            type="password"
            name="password"
            required
            minlength="3"
            maxlength="10"
            class="inputs"
          >
```
Закрывающая и открывающая дужка тега должны быть на одном уровне